src/visitor.rs

```diff
@@ -46,6 +46,85 @@ sfv::Parser::new(input).parse_item_with_visitor(&mut visitor)?;
 // Use `visitor.token` to do something expensive or with side effects now that
 // we know the entire input is valid.
 
+# Ok(())
+# }
+```
+
+Blanket implementations of [`ParameterVisitor`], [`ItemVisitor`],
+[`EntryVisitor`] and [`InnerListVisitor`] are provided for the corresponding
+`Option<V>` as a dynamic analogue of [`Ignored`]:
+
+```
+# use sfv::{BareItemFromInput, KeyRef, Parser, visitor::*};
+
+#[derive(Debug, Default, PartialEq)]
+struct Point {
+    x: i64,
+    y: i64,
+}
+
+struct CoordVisitor<'a> {
+    coord: &'a mut i64,
+}
+
+impl<'input> DictionaryVisitor<'input> for Point {
+    type Error = std::convert::Infallible;
+
+    fn entry<'dv, 'ev>(
+        &'dv mut self,
+        key: &'input KeyRef,
+    ) -> Result<impl EntryVisitor<'ev>, Self::Error>
+    where
+        'dv: 'ev,
+    {
+        let coord = match key.as_str() {
+            "x" => &mut self.x,
+            "y" => &mut self.y,
+            // Ignore all other keys
+            _ => return Ok(None),
+        };
+        Ok(Some(CoordVisitor { coord }))
+    }
+}
+
+#[derive(Debug)]
+struct NotAnInteger;
+
+impl std::fmt::Display for NotAnInteger {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        f.write_str("must be an integer")
+    }
+}
+
+impl std::error::Error for NotAnInteger {}
+
+impl<'input> ItemVisitor<'input> for CoordVisitor<'_> {
+    type Error = NotAnInteger;
+
+    fn bare_item<'pv>(
+        self,
+        bare_item: BareItemFromInput<'input>,
+    ) -> Result<impl ParameterVisitor<'pv>, Self::Error> {
+        if let BareItemFromInput::Integer(v) = bare_item {
+            *self.coord = i64::from(v);
+            Ok(Ignored)
```

Comment:
Does this make more sense at all?
```suggestion
            Ok(None::<Ignored>)
```

I wouldn't do that if I was writing this code, but this highlights the point you are making above.